resume/src/app/app.component.html

```diff
@@ -0,0 +1,18 @@
+<body class="bg-gray-light antialiased">
+<main class="p-5 max-w-5xl ml-auto mr-auto">
+  <div class="grid gap-5 lg:grid-cols-3">
```

Comment:
For the file structure, I will move this in a separate file (probably ResumeScreenComponent or so) and have only the routing tag (and navigation bar) in the app.component.html